docs/basics/integrations/webhooks.md

```diff
@@ -0,0 +1,80 @@
+---
+id: webhooks
+title: Webhooks Integration
+sidebar_label: Webhooks
+description: Setup a webhook to which Sauce Labs test results are automatically pushed.
+keywords:
+- get-test-results
+- webhooks
+- how-to
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+The Sauce Labs Webhooks integration allows you specify a URL to which Sauce Labs will push test result data as it becomes available, rather than forcing you to constantly poll for updates.
```

Comment:
At the moment we only send test result data, but in the future we may send different kind of data, for ex. credits used. Not sure if it is important right now.

Reply:
Yes. I had originally included concurrency, etc., but Lukasz thought it would be better to keep the doc specific to what is actually available now and update as we add features/functionality and I agree.